src/interfaces/uproxy_core_api.ts

```diff
@@ -290,5 +302,12 @@ export interface CoreApi {
   // This may also invoke an OAuth flow, in order to perform operations
   // with the cloud computing provider on the user's behalf.
   cloudInstall(args:CloudInstallArgs): Promise<void>;
+
+  // Invokes OAuth flow and destroys uProxy cloud server on the 
+  // provider specified.
+  cloudDestroy(providerName :string) : Promise<void>;
```

Comment:
@dborkan: Do we want to end up with many `cloudWhatever` calls? I can see `cloudUpdate`, `cloudRestart`, `cloudResize`, etc., appearing in the near future - and there's a lot of plumbing required for each.

What do you think about overloading `CloudInstallArgs`? (perhaps renaming it to CloudOperationArgs`)?